content/chainguard/chainguard-enforce/chainguard-enforce-signing/verify-enforce-signed-images.md

```diff
@@ -0,0 +1,82 @@
+
+---
+title: "Example Policy for Enforce Signed Images"
+type: "article"
+description: "Verify images signed by Chainguard Enforce"
+date: 2022-07-15T15:22:20+01:00
+lastmod: 2022-11-29T15:22:20+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-enforce-signing"
+weight: 100
+toc: true
+---
+
+Chainguard Enforce for Kubernetes allows users to create their own security policies that they can enforce in their clusters. 
+
+Here is an example of a policy template used to verify images that has been signed by our Enforce signing feature.
+
+```yaml
+apiVersion: policy.sigstore.dev/v1beta1
+kind: ClusterImagePolicy
+metadata:
+  name: enforce-signed-keyless
+spec:
+  images:
+  - glob: image/*
+  authorities:
+  - keyless:
+      trustRootRef: my-sigstore-keys
+      url: https://<my-enforce-sigstore-hostname-instance>
+      identities:
+        - issuer: 'https://auth.chainguard.dev/'
+          subject: 'foo@example.com'
+      insecureIgnoreSCT: true # In Sigaas, there isn't any proof of inclusion in a certificate transparency log.
```

Comment:
Would "Sigaas" make sense for customers? Should we use "Enforce Signing" or something else?

Reply:
Yeah, we better use Enforce signing.